rs/state_machine_tests/src/lib.rs

```diff
@@ -2649,7 +2649,14 @@ impl StateMachine {
         .0;
 
         let (h, mut state) = self.state_manager.take_tip();
+
+        // Repartition input schedules; Required step for migrating canisters.
+        canister_state
+            .system_state
+            .split_input_schedules(&canister_id, &state.canister_states);
```

Comment:
Are we sure we want to split input schedules as part of importing canister state? If so we should at least document that we expect it to be a no-op otherwise.

Reply:
I guess this function is only used in the context of migration anyways... so updating the documentation of the function might be fine as well.

Reply:
Thanks for clarifying. Makes sense -- I overlooked that it is a noop if not related to a migration.